implementations/rust/ockam/ockam_command/README.md

```diff
@@ -8,6 +8,8 @@
 Orchestrate end-to-end encryption, mutual authentication, key management,
 credential management, and authorization policy enforcement — at scale.
 
+This crates provides the `ockam` command line application to start Ockam nodes and interact with them.
```

Comment:
Since the CLI is not limited to work with nodes, maybe we could add more information here, similar to what we have here: https://docs.ockam.io/reference/command

Reply:
Thanks @adrianbenavides I expanded slightly the description and then put a link the the command doc directly, since I don't want to duplicate the information that is already there.

I'm putting back this PR as a draft until I can fix the failing CI job(s) and I have added a CI check to make sure that the readmes are up to date.